samples/cpp/prompt_lookup_decoding_lm/CMakeLists.txt

```diff
@@ -6,16 +6,29 @@ if(TARGET openvino_tokenizers)
 else()
     message(FATAL_ERROR "multinomial_causal_lm must be compiled as part of OpenVIINOGenAI to have the path to openvino_tokenizers hardcoded.")
 endif()
+
+include(FetchContent)
+
+FetchContent_Declare(nlohmann_json
+    URL https://github.com/nlohmann/json/archive/refs/tags/v3.11.3.tar.gz
+    URL_HASH SHA256=0d8ef5af7f9794e3263480193c491549b2ba6cc74bb018906202ada498a79406)
+FetchContent_MakeAvailable(nlohmann_json)
```

Comment:
please, add the similar guards for nlohmann as in https://github.com/openvinotoolkit/openvino.genai/pull/726 and define CMP policy 135